MerlinAU.sh

```diff
@@ -75,7 +76,8 @@ cronCmd="$(which crontab) -l"
 ##----------------------------------------------##
 inMenuMode=true
 isInteractive=false
-menuReturnPromptStr="Press Enter to return to the main menu..."
```

Comment:
You know, I'm not sure why I didn't go this route the first time. Seems much less work to only have to update it once instead of everywhere like you had too today lol. Approved!

Reply:
Tested and working.